plugins/modules/eval_nat_network_acls.py

```diff
@@ -65,40 +65,26 @@
     dst_ip: "8.8.8.8"
     dst_port: 80
     nat_network_acls:
-        - egress:
-            - - 100
-              - "all"
-              - "allow"
-              - "0.0.0.0/0"
-              - null
-              - null
-              - 0
-              - 65535
-        - ingress:
-            - - 100
-              - "all"
-              - "allow"
-              - "0.0.0.0/0"
-              - null
-              - null
-              - 0
-              - 65535
+    - egress:
+      - [100, "all", "allow", "0.0.0.0/0", null, null, 0, 65535]
```

Comment:
Have you transformed a 'list of lists' into a 'list of strings'? Is my understanding correct, and if so, could you explain the reason behind this change? It seems like the data type hasn't been altered in [this context.](https://github.com/redhat-cop/cloud.aws_troubleshooting/pull/31/files#diff-b1ff93fefcd354c29e0a57011bc15d03a86636b791bfc80614fd26675db96a77L74)

Reply:
@GomathiselviS it is exactly the same, a list of list of string, but I will turn it back as it was before to avoid confusion